build-logic/profiling/src/main/kotlin/gradlebuild.buildscan.gradle.kts

```diff
@@ -99,30 +99,32 @@ fun isMonitoredAsciidoctorTask() = false // No asciidoctor tasks are cacheable f
 fun isExpectedAsciidoctorCacheMiss() =
 // Expected cache-miss for asciidoctor task:
 // 1. CompileAll is the seed build for docs:distDocs
-// 2. Gradle_Check_BuildDistributions is the seed build for other asciidoctor tasks
+// 2. BuildDistributions is the seed build for other asciidoctor tasks
 // 3. buildScanPerformance test, which doesn't depend on compileAll
-    // 4. buildScanPerformance test, which doesn't depend on compileAll
+// 4. buildScanPerformance test, which doesn't depend on compileAll
     isInBuild(
-        "Gradle_Check_CompileAll",
-        "Gradle_Check_BuildDistributions",
-        "Enterprise_Master_Components_GradleBuildScansPlugin_Performance_PerformanceLinux",
-        "Enterprise_Release_Components_BuildScansPlugin_Performance_PerformanceLinux"
+        "Check_CompileAll",
+        "Check_BuildDistributions",
+        "Enterprise_Master_Component_GradlePlugin_Performance_PerformanceLatestMaster",
```

Comment:
We should drop the `Enterprise_Master_` prefix here as well.

Reply:
Done, PATL!